app/controllers/phrasing_phrases_controller.rb

```diff
@@ -6,7 +6,7 @@ class PhrasingPhrasesController < Phrasing.parent_controller.constantize
 
   include PhrasingHelper
 
-  before_filter :authorize_editor
+  before_action :authorize_editor
```

Comment:
Rails 5 finally doesn't support before_filters (switched completely to before_actions)?